Android.mk

```diff
@@ -1,3 +1,17 @@
+# Copyright 2018 The Amber Authors.
```

Comment:
2019

Reply:
I though these ones were dating back to 2018 :) Fixed!